artio-codecs/src/test/java/uk/co/real_logic/artio/util/DecimalFloatEncodingTest.java

```diff
@@ -36,27 +36,92 @@
         return Arrays.asList(new Object[][]
         {
             {"55.36", 5536L, 2},
-            {".995", 995L, 3},
+            {"0.995", 995L, 3},
             {"25", 25L, 0},
             {"-55.36", -5536L, 2},
-            {"-.995", -995L, 3},
+            {"-0.995", -995L, 3},
             {"-25", -25L, 0},
             {"1.1", 11L, 1},
-            {".6", 6L, 1},
-            {".06", 6L, 2},
-            {"-.6", -6L, 1},
-            {"-.06", -6L, 2},
+            {"0.6", 6L, 1},
+            {"0.06", 6L, 2},
+            {"-0.6", -6L, 1},
+            {"-0.06", -6L, 2},
             {"10", 10L, 0},
             {"-10", -10L, 0},
 
-            {".92117125", 92117125L, 8},
-            {"-.92117125", -92117125L, 8},
-            {".92125", 92125L, 5},
-            {"-.92125", -92125L, 5},
-            {".00007875", 7875, 8},
-            {"-.00007875", -7875, 8},
+            {"0.92117125", 92117125L, 8},
+            {"-0.92117125", -92117125L, 8},
+            {"0.92125", 92125L, 5},
+            {"-0.92125", -92125L, 5},
+            {"0.00007875", 7875, 8},
+            {"-0.00007875", -7875, 8},
             {"1.00109125", 100109125, 8},
             {"-1.00109125", -100109125, 8},
+
+            // negative scale
+            {"2000", 2L, -3},
+            {"-2000", -2L, -3},
+
+            // edge values
+            {"0", 0, 0},
+            //{"0.00", 0, 2},
```

Comment:
Why is this value commented out? Either it should pass or if it isn't a valid value then it should be removed.

Reply:
I would just remove this entry. A value of 0 is always equal to zero and there's point in printing out "0.00" when "0" would suffice.

Reply:
ok removed it.

Reply:
ok, removed it.